edx_solutions_api_integration/users/tests.py

```diff
@@ -50,7 +47,13 @@
 from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
 from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase, mixed_store_config, SharedModuleStoreTestCase
 
-from django.contrib.auth.models import User
+from edx_solutions_api_integration.test_utils import (
+    get_non_atomic_database_settings,
+    CourseGradingMixin,
+    APIClientMixin,
+    SignalDisconnectTestMixin,
+)
+
```

Comment:
The changes on these imports looks like a mess.  All that's really happening is I cleaned up the sorting, and added `import six`.

Reply:
👍